jenkins/config.py

```diff
@@ -12,6 +12,7 @@
                 "PRIMER_DETECTED",
                 "PRIMER_NUMREADS",
                 "PRIMER_UNIQUE_NUMREADS",
+                "PRIMER_AMBIGUOUS_NUMREADS",
```

Comment:
This is a new primer column which counts the number of ambiguous primer sequences found in the sample reads. These sequences contain `N`s.